src/js/options.ts

```diff
@@ -206,6 +206,11 @@ export interface BaseReactNativeOptions {
    */
   enableNativeFramesTracking?: boolean;
 
+  /**
+   * Track when and how long the JS event loop stalls for. Adds stalls as measurements to all transactions.
+   */
```

Comment:
NIT: It's nice to have the default value on the comments so it doesn't generate doubts with users.

```suggestion
   *
   * @default true
   */
```